omod/src/main/resources/config.xml

```diff
@@ -16,6 +16,10 @@
 		<require_module version="${openmrsAtomfeedVersion}">org.ict4h.openmrs.openmrs-atomfeed</require_module>
 	</require_modules>
 
+	<aware_of_modules>

```

Comment:
We need to add to this:

```xml
<conditionalResources>
	<conditionalResource>
		<path>/lib/fhir2-appointment-api-${project.parent.version}.jar</path>
		<loadIfModulesPresent>
			<openmrsModule>
				<moduleId>org.openmrs.module.fhir2</moduleId>
				<version>1.*</version>
			</openmrsModule>
		</loadIfModulesPresent>
	</conditionalResource>
</conditionalResources>
```

So that the FHIR-specific stuff is only loaded if the FHIR module is present.

Reply:
First, you will have to upgrade the `config.xml `DTD file version from 1.2 to 1.6
Either this
```java
<!DOCTYPE module PUBLIC "-//OpenMRS//DTD OpenMRS Config 1.0//EN"
		"https://resources.openmrs.org/doctype/config-1.6.dtd">
```
or 
`<module configVersion="1.6">`
This is because` loadIfModulesPresent` was introduced in version 1.6 Also, `conditionalResources`